met-api/app/services/engagement_service.py

```diff
@@ -0,0 +1,30 @@
+from app.models.engagement import Engagement
+
+class engagement_service:
+    """ Engagement management service
+    """
+    otherdateformat = '%Y-%m-%d'
+
+    def get_engagement(self, engagement_id):
+        request_engagement = Engagement().get_engagement(engagement_id)
```

Comment:
I'm not familiar with python but does Engagement() means its creating a new instance of the engagement class?

Reply:
Good eye, I have changed it to Engagement without calling it. Bit of research is perhaps needed here to make sure whether  python creates an Engagement instance at import and not interprets Engagement.get_engagement same as Engagement().get_engagement